subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,8 +1,22 @@
-The Gradle team is pleased to announce the release of Gradle 4.1.
+The Gradle team is pleased to announce the first release candidate for Gradle 4.1.
```

Comment:
Reverted, but I'd like to keep this RC language for GitHub because the releases are separate.